code/__DEFINES/dcs/signals/signals_datum.dm

```diff
@@ -31,3 +31,17 @@
 #define COMSIG_MERGER_REMOVING "comsig_merger_removing"
 /// Called on the merger after finishing a refresh: (list/leaving_members, list/joining_members)
 #define COMSIG_MERGER_REFRESH_COMPLETE "comsig_merger_refresh_complete"
+
+// Gas mixture signals
+/// Called when a gasmix have air added into it.
+#define COMSIG_GASMIX_MERGING "comsig_gasmix_merged"
+/// Called when a gasmix have air being subtracted from it.
+#define COMSIG_GASMIX_REMOVING "comsig_gasmix_removing"
+/// Called when a gasmix is currently reacting.
+#define COMSIG_GASMIX_REACTING "comsig_gasmix_reacting"
```

Comment:
These comments aren't completely true, should fix that. You should also follow the standard context of source proc : (args)
Does most of the talking for you